includes/class-wc-payments-order-service.php

```diff
@@ -1364,7 +1370,7 @@ private function update_order_status( $order, $order_status, $intent_id = '' ) {
 	 *
 	 * @return array The data we need to continue processing.
 	 */
-	private function get_intent_data( $intent ): array {
+	public function get_intent_data( $intent ): array {
```

Comment:
This method is a hack due to intents can be objects or arrays and is meant to be private for that reason, it shouldn't be used outside of this class. When I added it previously I should have been clearer in the docblock.

Reply:
Addressed on [c383b20](https://github.com/Automattic/woocommerce-payments/pull/5900/commits/c383b200be22ea794cf4d3465c337e4f97581b6c).